docs/docs/reference-react-dom.md

```diff
@@ -0,0 +1,70 @@
+---
+id: reference-react-dom
+title: ReactDOM
+permalink: docs/reference-react-dom.html
+next: reference-react-dom-server.html
+redirect_from: "/docs/reference.html"
+---
+
+The `react-dom` package provides DOM-specific methods that can be used at the top level of your app and as an escape hatch to get outside of the React model if you need to. Most of your components should not need to use this module.
+
+ - [`render()`](#render)
+ - [`unmountComponentAtNode()`](#unmountComponentAtNode)
+ - [`findDOMNode()`](#findDOMNode)
+
+## Reference
+
+### `render()`
+
+```javascript
+ReactDOM.render(
+  element,
+  container,
+  [callback]
+)
+```
+
+Render a ReactElement into the DOM in the supplied `container` and return a [reference](/react/docs/more-about-refs.html) to the component (or returns `null` for [stateless components](/react/docs/reusable-components.html#stateless-functions)).
```

Comment:
Can this link to new doc instead? "Components and Props" includes a section on functional components. Also let's call them "functional components" since that's their defining trait here. Class components can also be stateless but this is not important here.